netlify.toml

```diff
@@ -12,7 +12,7 @@
   command = "go run mage.go -v Docs"
 
 [build.environment]
-  HUGO_VERSION = "0.115.4"
+  HUGO_VERSION = "0.116.1"
```

Comment:
This looks good and thank you for fixing this issue. Can you update the Hugo version in the CONTRIBUTING.md as well.

Reply:
Done thanks